apps/catalogue/src/components/RefbackTable.vue

```diff
@@ -124,20 +124,25 @@ export default {
     };
   },
   methods: {
-    convertToPascalCase,
     handleRowClick(row) {
+      let params = {};
+      params.id = row.id ? row.id : this.pkey.id;
```

Comment:
Minor thingy: in js you'd usually use the 'row.id || this.pkey.id' shorthand here instead of a ternary :)

Reply:
ah, this is my java background. The js syntax is so much nicer